controllers/auth.controller.js

```diff
@@ -0,0 +1,22 @@
+const { Auth } = require('../entities/auth.entities');
+
+const storeInstallation = async (installation) => {
+  if (installation.team !== undefined) {
```

Comment:
is it possible that installation.team equals to null?

Reply:
This validation is provided by the documentation so I suppose that it returns a value or undefined.

Reply:
ok